memcnn/models/tests/test_split_dim.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+
+import pytest
+import torch
+
+from memcnn import AdditiveCoupling, AffineCoupling, InvertibleModuleWrapper
+
+
+class Check(torch.nn.Module):
+    def __init__(self, dim, target_size):
+        super(Check, self).__init__()
+        self.dim = dim
+        self.target_size = target_size
+
+    def forward(self, fn_input: torch.Tensor) -> torch.Tensor:
```

Comment:
You can't use typing arguments here since we also support old Python versions (2.7). 
This also shows up in the CI tests.

Reply:
You're right. I'll have to remove them. Unfortunately I cant view the CI information as CircleCI kicks me out of your instance immediately.